kornia/nerf/nerf_solver.py

```diff
@@ -143,17 +144,17 @@ def _train_one_epoch(self) -> float:
         ray_dataset.init_ray_dataset(self._num_img_rays)
         ray_dataset.init_images_for_training(self._imgs)  # FIXME: Do we need to load the same images on each Epoch?
         ray_data_loader = instantiate_ray_dataloader(ray_dataset, self._batch_size, shuffle=True)
-        total_psnr = 0.0
+        total_psnr = torch.tensor(0.0, device=self._device, dtype=self._dtype)
```

Comment:
we have exposed `from kornia.core import tensor` -- at some point we'll have our own tensor/ndarray so better to use from core as much as possible

Reply:
Done on [35eebe6](https://github.com/kornia/kornia/pull/1967/commits/35eebe6a10eb3f8a6f7eb65048e2840405096d99)